### PR TITLE
cdt-cc man page created

### PR DESCRIPTION
**Pull request by jolly-fellow**
## API Changes
- [ ] API Changes

## Documentation Additions
- [ x] Documentation Additions

### CODE REVIEW

File: docs/man/cdt-cc.1.md [hunk 1]
```diff
@@ -0,0 +1,351 @@
+---
+title: CDT-CC
+section: 1
+header: User Manual
```
Comment (ScottBailey) [inline docs/man/cdt-cc.1.md:4]: I think this is a better header?
```suggestion
header: Contract Development Toolkit (CDT)
```

Comment (ScottBailey) [reply docs/man/cdt-cc.1.md:4]: It looks like cdt-cpp.1.md has the same issue, I think we can fix it here as well.

**Review by ScottBailey — CHANGES_REQUESTED** [review]
Please fix header.

**Review by mikelik — APPROVED** [review]

File: docs/man/cdt-cc.1.md (hunk 1 above)
Comment (jolly-fellow) [reply docs/man/cdt-cc.1.md]: > It looks like cdt-cpp.1.md has the same issue, I think we can fix it here as well.

Updated the headers here and in cdt-cpp

**Review by ScottBailey — APPROVED** [review]

**Pull request merged**